website/docs/reference/opspec/index.md

```diff
@@ -101,5 +34,55 @@ Reference:
 - [socket](types/socket.md)
 - [string](types/string.md)
 
-## Scoping
-Variables in opspec are scoped to each operation. [Parameters](../op/parameter/index.md) allow passing values into or out of this scope.
\ No newline at end of file
+### Scoping
+
+A variable is created when defined as an [`input`](op.yml#inputs) or [`output`](op.yml#inputs), bound implicitly to an `output` when making an [op call](op.yml/call/op), or declared with [loop variables](./op.yml/loop-variable). A variable name is in scope from creation through the lifetime of it's `op.yml` file. Variable scope does not extend into [op calls](./op.yml/call/op).
```

Comment:
This reads ambiguously to me `Variable scope does not extend into [op calls](./op.yml/call/op)`. Is it the current op call, child op call?

Reply:
The link here exists to make unambiguous. Literally, `op` calls in the `op.yml`. I'll reword though